### ISSUE TITLE
loop_var output is wrong

### ISSUE THREAD

**Issue by nerzhul** [ansible/ansible#16144]
##### ISSUE TYPE
- Bug Report
##### ANSIBLE VERSION

```
ansible 2.1.0.0
```
##### CONFIGURATION

Not important there
##### OS / ENVIRONMENT

Archlinux / Debian 8
##### SUMMARY

When iterating over multiple with_items loop, we should use loop_control.loop_var variable to iterate over the multiple levels.
Ansible output is wrong on screen
##### STEPS TO REPRODUCE

Create a role including a file using with_items
Do a task with a with_item inside it

main.yml

```
- include: deploy_single.yml app_item="{{ item }}"
  with_items: "{{ php_apps }}"
```

deploy_single.yml

```
- name: "FILE | Apply httpd rights for application"
  file: path="{{ dir_item }}" state=directory recurse=yes owner="{{ app_item.httpd_write_owner | default('www-data') }}"
  with_items: "{{ app_item.httpd_write_dirs }}"
  loop_control:
    loop_var: dir_item
```
##### EXPECTED RESULTS

```
TASK [php_application : TEMPLATE | Deploy templates] ***************************
ok: [test.local] => (tpl_item="/var/www/owncloud/data")
```
##### ACTUAL RESULTS

```
TASK [php_application : TEMPLATE | Deploy templates] ***************************
ok: [test.local] => (item=None)

```

**Comment by kamsz** [comment]
confirmed

**Comment by jimi-c** [comment]
Hi @nerzhul, just following up on some older tickets and I see this was resolved in 2.2.0, so closing it now.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:

   * https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
   * https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular issue is resolved.

Thank you!
